android/src/test/java/org/mozilla/mozstumbler/client/UpdaterTest.java

```diff
@@ -59,16 +62,14 @@ public void testUpdater() {
         ServiceLocator.getInstance().putService(IHttpUtil.class, mockHttp);
         ServiceLocator.getInstance().putService(ISystemClock.class, clock);
 
-        // wifi is always unavailable
+        // network is always available
         doReturn(false).when(upd).wifiExclusiveAndUnavailable(Mockito.any(Context.class));
 
         assertFalse(upd.checkForUpdates(activity, ""));
-        now += Updater.UPDATE_CHECK_FREQ_MS;
```

Comment:
I moved the `sLastUpdateCheck` assertions to `testUpdaterThrottlesRequests`.
Also, if the last call in this test now successfully returns true, then I think one can assume that the previous calls did not have any side effects.